### PR TITLE
style: update comments DNA-15944 (DNA-19979)

### PR DESCRIPTION
**Pull request by JasperSelmanUiPath**
## Description
- Update the comments in the macros to put them next to the correct statements.
- Move `id` macro to the correct folder.
- Fixed `to_date` macro to be consistent with Snowflake
- Fixed `to_timestamp` macro to parse milliseconds by default.

## Release
- [ ] Direct release (`main`)
- [x] Merge to `dev` (or other) branch
  - Why: will be released with the databricks track.

### Did you consider?
~~- [ ] Does it Work on Automation Suite / SQL Server~~
~~- [ ] Does it Work on Automation Cloud / Snowflake~~
~~- [ ] What is the performance impact?~~
- [x] The version number in `dbt_project.yml`~~
  - will be released with the databricks track.

### CODE REVIEW

**Review by cverhoef — COMMENTED** [review]
And also update the readme when moving the id function to a different folder.

File: macros/multiple_databases/to_date.sql [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {%- macro to_date(field, relation) -%}
 
-{# Cast to date when the input is in a date or a datetime format. This is default behavior for SQL Server. #}
 {%- if target.type == 'databricks' -%}
     to_date({{ field }}, '{{ var("date_format", "yyyy-MM-dd") }}')
+{# Cast to date when the input is in a date or a datetime format. This is default behavior for SQL Server. #}
```
[the remark this answers is not in the document]
Comment (JasperSelmanUiPath) [reply macros/multiple_databases/to_date.sql]: - Fixed the comment.
- Tested the behavior of the macro and the Databricks implementation of this macro cannot deal with datetime input in its current implementation. Created https://uipath.atlassian.net/browse/DNA-20072 to fix it.

File: README.md [hunk 2]
```diff
@@ -146,7 +146,7 @@ Usage:
 `{{ pm_utils.to_integer('[expression]') }}`
 
 #### to_timestamp ([source](macros/multiple_databases/to_timestamp.sql))
-This macro converts a field to a timestamp field.
+This macro converts a field to a timestamp field. For Databricks, milliseconds will automatically be parsed. You should only set the milliseconds in your datetime patern if the second fraction is different than milliseconds.
```
Comment (cverhoef) [inline README.md:149]: Is this correct or the example above?
So this: `yyyy-MM-dd HH:mm:ss.SSS` or `yyyy-MM-dd HH:mm:ss`?

Comment (silviustanimir) [reply README.md:149]: `yyyy-MM-dd HH:mm:ss` I see what you meant, I'll fix the README.md

File: macros/multiple_databases/to_date.sql [hunk 3]
```diff
@@ -2,7 +2,11 @@
 
 {# Cast to date when the input is in a date or a datetime format. This is default behavior for SQL Server. #}
 {%- if target.type == 'databricks' -%}
-    to_date({{ field }}, '{{ var("date_format", "yyyy-MM-dd") }}')
+    case
+        when to_date({{ field }}, '{{ var("date_format", "yyyy-MM-dd") }}') is null
```
Comment (Dennis-UiPath) [inline macros/multiple_databases/to_date.sql:6]: For which case do we need this condition? In most cases (combined with legacy setting) the parsing will work. When parsing will fail (and the result will be null), the to_date function will also fail, but I might miss a usecase?

Comment (silviustanimir) [reply macros/multiple_databases/to_date.sql:6]: This is done in line with Snowflake, the "extra" use case is to try parsing the input string to a date without giving an explicit date format, which is optional. By chance, the default is for now the same, but say that this changes in an UDF, then we would get an extra fallback.
I do agree that for now it doesn't bring much added value.

**Review by Dennis-UiPath — APPROVED** [review]

**Pull request merged**